enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/EagerizationAcceptanceTest.scala

```diff
@@ -238,7 +262,15 @@ class EagerizationAcceptanceTest
     createNode()
     val query = "MATCH (a), (b) CALL user.mkRel(a, b) MATCH (a)-[rel]->(b) RETURN rel"
 
-    val result = executeWithCostPlannerAndInterpretedRuntimeOnly(query)
+    // only run against one scenario, otherwise count increments unpredictably due to fallback
+    val succeedScenarios = Configs.Version3_3 - Configs.Procs - Configs.Compiled - Configs.AllRulePlanners - Configs.SlottedInterpreted
```

Comment:
I'm wondering if we could specify one scenario explicitly using the new API? E.g. `TestConfiguration(3.3, Cost, Interpreted)`